internal/binarylog/method_logger_test.go

```diff
@@ -336,7 +338,9 @@ func (s) TestLog(t *testing.T) {
 	for i, tc := range testCases {
 		buf.Reset()
 		tc.want.SequenceIdWithinCall = uint64(i + 1)
-		ml.Log(context.Background(), tc.config)
+		ctx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
+		defer cancel()
```

Comment:
Let's move this outside the loop?

Reply:
Sure, moved context creation to the outside.